crates/e2e/tests/e2e/refunder.rs

```diff
@@ -0,0 +1,189 @@
+use crate::services::{deploy_mintable_token, to_wei, OrderbookServices, API_HOST};
+use ethcontract::{Account, Address, Bytes, H256, U256};
+use model::{
+    order::{OrderBuilder, OrderKind},
+    quote::{OrderQuoteRequest, OrderQuoteResponse, OrderQuoteSide, QuoteSigningScheme, Validity},
+    signature::hashed_eip712_message,
+    DomainSeparator,
+};
+use refunder::refund_service::{EncodedEthflowOrder, RefundService, INVALIDATED_OWNER};
+use shared::{ethrpc::Web3, http_client::HttpClientFactory, maintenance::Maintaining};
+use sqlx::PgPool;
+
+const QUOTING_ENDPOINT: &str = "/api/v1/quote/";
+
+#[tokio::test]
+#[ignore]
+async fn local_node_smart_contract_orders() {
```

Comment:
This identifier is used to select the test to run. Those identifiers should have a unique and descriptive name.
Right now this one is duplicated.